doc/classes/OS.xml

```diff
@@ -384,7 +384,7 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the absolute directory path where user data is written ([code]user://[/code]).
+				Returns the absolute directory path where user data is written ([code]user://[/code]) if the project has a name. Returns the absolute resource path otherwise.
```

Comment:
It's good to mention it, but I would phrase it differently. Here it sounds like a weird fallback, while it actually *uses* `res://` as the `user://` if no project name is defined (so it's not just `OS.get_user_data_dir()`, it's the whole `user://` logic).

It might be better to actually fully document what `user://` resolves to here?